client/my-sites/customer-home/cards/features/help-search/style.scss

```diff
@@ -157,23 +157,23 @@ $min_results_height: 175px;
 	// PlaceholderLines component
 	.inline-help__results-placeholder {
 		min-height: $min_results_height;
-		margin: $search_results_top_spacing 0 0;
+		margin: #{$search_results_top_spacing + 1} #{-$search_results_top_spacing} 0;
```

Comment:
It seems that
`margin: #{$search_results_top_spacing + 1} 0 0 #{-$search_results_top_spacing};`
would be better in order to avoid animating placeholders touching the container on the right?
![SS 2020-06-30 at 14 30 41](https://user-images.githubusercontent.com/12430020/86121233-59c7e500-bade-11ea-9179-3e3266f99ba2.jpg)